security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,44 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-16-0_{context}"]
+== {cert-manager-operator} 1.16.0
+
+Issued: 2025-05-27
+
+The following advisories are available for the {cert-manager-operator} 1.16.0:
+
+* link:https://access.redhat.com/errata/RHEA-2025:8163[RHEA-2025:8163]
+* link:https://access.redhat.com/errata/RHEA-2025:8164[RHEA-2025:8164]
+* link:https://access.redhat.com/errata/RHEA-2025:8165[RHEA-2025:8165]
+* link:https://access.redhat.com/errata/RHEA-2025:8198[RHEA-2025:8198]
+
+Version `1.16.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.16.4`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.16/#v1164[cert-manager project release notes for v1.16.4].
+
+[id="cert-manager-operator-1-16-0-features-enhancements_{context}"]
+=== New features and enhancements
+
+*Disconnected installation support*
+
+With this release, the {cert-manager-operator} can be installed in a disconnected environment with the following issuer types: ACME, CA, Self-signed, and Vault.
+Specifically, private or self-hosted ACME servers have been validated, as Let's Encrypt or other public ACME services are not feasible options in disconnected environments.
+To mirror Operator images, the preferred method is using the oc-mirror plugin v2.
+For more information, see xref:../../disconnected/mirroring/about-installing-oc-mirror-v2.adoc#about-installing-oc-mirror-v2[Mirroring images for a disconnected installation by using the oc-mirror plugin v2].
+
+*Extended operand metrics support*
+
+With this release, cert-manager webhook and cainjector operands now expose Prometheus metrics on port 9402 by default via the `/metrics` service endpoint.
+You can configure OpenShift Monitoring to collect metrics from all cert-manager operands by enabling the built-in user workload monitoring stack.
+For more information, see xref:../../observability/monitoring/configuring-user-workload-monitoring/preparing-to-configure-the-monitoring-stack-uwm.adoc#preparing-to-configure-the-monitoring-stack-uwm[Preparing to configure the user workload monitoring stack].
```

Comment:
While this looks great, we might just need an additional follow-up change to update the metrics page with the new cainjector and webhook metrics details.

Reply:
Updated the xref to link to the cert manager specific page for monitoring.

As for updating that content, I can help if Subhashini does not come back in time, otherwise I will leave that update to her.